src/api/prompt.ts

```diff
@@ -466,9 +466,13 @@ export function notifyPromptActionRequested(action: "next" | "previous" | "clear
 }
 
 /**
- * Awaits a keypress from the user and returns the entered key.
+ * Awaits for one or more keypresses from the user and returns the entered keys.
+ *
+ * @param keyCount Determines the number of keypresses to wait for; defaults to 1.
  */
-export async function keypress(context = Context.current): Promise<string> {
+export async function keypress(params: Readonly<{ keyCount?: number }>,
+                               context = Context.current): Promise<string> {
+  const keyCount = params.keyCount || 1;
```

Comment:
If `params.keyCount` is `0`, this will evaluate to `1`. I'm not sure how I feel about that. IMO for `params.keyCount === 0` we could return immediately with an empty string.